library/Fission/CLI/Auth.hs

```diff
@@ -47,3 +61,4 @@ withAuth action = get >>= \case
     liftIO $ ANSI.setSGR [ANSI.SetColor ANSI.Foreground ANSI.Vivid ANSI.Blue]
     UTF8.putText "fission-cli login"
     liftIO $ ANSI.setSGR [ANSI.Reset]
+    return . Left . CLIError . UTF8.textShow $ err
```

Comment:
Looks like we use `return . Left . CLIError . UTF8.textShow $ err` twice in this function; it may be a sign that this is a reusable chunk:

```haskell
foo = Left . CLIError . UTF8.textShow
```